utils/python/CIME/XML/env_archive.py

```diff
@@ -3,14 +3,27 @@
 """
 from standard_module_setup import *
 
-from env_base import EnvBase
```

Comment:
env_archive does not follow the entry id format and so cannon inherit from EnvBase.